.github/workflows/linting.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Code Linting
+on:
+  pull_request:
+  push:
```

Comment:
add workflow_dispatch so we can test manually?

Reply:
Done https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pv/pull/1/commits/b450e352c1dbdda8c9c579389c6e6eed4532cb34

@proppy